pkg/macos/systempreferences/dock.go

```diff
@@ -70,20 +130,14 @@ func getAppPath(appName string) (string, error) {
 	return "", fmt.Errorf("Could not find path for app %s", appName)
 }
 
-func killDock() {
-	cmd := exec.Command("killall", "Dock")
-
-	err := cmd.Run()
-	if err != nil {
-		log.Fatalf("cmd.Run() failed with %s\n", err)
-	}
+func killDock() error {
+	return exec.Command("killall", "Dock").Run()
 }
 
-func boolToInt(b bool) int {
-	var bitSetVar int
-	if b {
-		bitSetVar = 1
+func boolToInt(booleanValue bool) int {
```

Comment:
Interesting function, since Go is typed you can always just return `booleanValue` and not do an `if`.